### PR TITLE
Add Web/API page types, part 13

### PR DESCRIPTION
**Pull request by wbamberg**
See  https://github.com/mdn/content/issues/16255. This is part 13 of a series of PRs to add page types to the Web/API documentation.

### CODE REVIEW

**Review by estelle — APPROVED** [review]

File: files/en-us/web/api/indexeddb/index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: indexedDB
 slug: Web/API/indexedDB
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/indexeddb/index.md:4]: This is a global property, not an interface.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
One error I think. 

Good catch, for the 4 static methods on `IDBKeyRange`. There are no mention of it in the articles. We probably want, once we have page types all over APIs, to be sure static methods are documented (in the prose) as such.

File: files/en-us/web/api/indexeddb/index.md (hunk 1 above)
Comment (wbamberg) [reply files/en-us/web/api/indexeddb/index.md]: Ooh, good catch, thank you!

Comment (wbamberg) [reply files/en-us/web/api/indexeddb/index.md]: and yes, at the moment we don't have any reliable way to indicate statics AFAIK.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**